tests/test_pep518.py

```diff
@@ -0,0 +1,29 @@
+import subprocess
+import sys
+
+import pytest
+
+from .conftest import cd_package
+
+
+@pytest.mark.parametrize(
+    ('package'),
+    [
+        'scipy-like',
+    ]
+)
+@pytest.mark.parametrize(
+    'build_arg', ['', '--wheel'], ids=['sdist_to_wheel', 'wheel_directly']
+)
+def test_pep518(pep518, package, build_arg, tmp_path):
+    dist = tmp_path / 'dist'
+
+    try:
+        import build  # noqa: F401
+    except ModuleNotFoundError:
+        print('Installing build for testing')
+        subprocess.run([sys.executable, '-m', 'pip', 'install', 'build'], check=True)
```

Comment:
Please don't do this. I don't want the tests to touch my Python environment. You added `build` to the extra-requires and it is sufficient.

Reply:
Oops, I thought someone else added this, and I see I did - it was just originally in a venv (where this made sense), and now it's not. Will remove.